reference/5.1/Microsoft.PowerShell.Core/About/about_Functions.md

```diff
@@ -144,17 +153,17 @@ for named parameters, as described later in this topic.
 You can define parameters inside the braces using the `Param` keyword, as
 shown in the following sample syntax:
 
-```
+```syntax
 function <name> {
-  param ([type]$parameter1[,[type]$parameter2])
+  param ([[type]$parameter1,[type]$parameter2])
```

Comment:
The second parameter is optional too. So this is the correct syntax.

```suggestion
  param ([[type]$parameter1 [,[type]$parameter2]])
```

It is confusing either way.